### ISSUE TITLE
workerPort option not working anymore after 21.1.2 release

### ISSUE THREAD

**Issue by jbaan** [stephanrauh/ngx-extended-pdf-viewer#2540]
I wanted to upgrade my pdf viewer application from 21.1.2 towards 21.3.x release, but then the pdf viewer is not working anymore. In the console I see that the system is trying to setup a fakeworker. I debugged the issue and found location which is wrong

in the viewer-x.mjs file in the function

```js
 async open(args) {
    this.pdfViewer?.destroyBookMode();
    if (this.pdfLoadingTask) {
      await this.close();
    }
    const workerParams = AppOptions.getAll(OptionKind.WORKER);
    if (args.workerSrc && args.workerSrc !== workerParams.workerSrc) {
      workerParams.workerSrc = args.workerSrc;
      this.pdfWorker = null;
    }
    Object.assign(GlobalWorkerOptions, workerParams);
    if (this.pdfWorker === null) {
      this.pdfWorker = workerParams.port ? PDFWorker.fromPort(workerParams) : new PDFWorker(workerParams);
    }
```

There is a check done: `workerParams.port`, but this value is not there, the `workerParams.workerPort` is now set. And therefore `PDFWorker.fromPort` is skipped and system is trying to setup a fakeWorker... 

So `workerParams.port` should be set with value of `workerParams.workerPort`.

**Comment by stephanrauh** [comment]
Sounds like a thorough analysis, so I'll accept the bug without having reproduced it. Nonetheless, I'd like to reproduce the bug. Why does it happen in your project, but not on https://pdfviewer.net/extended-pdf-viewer/simple? Can you spot a difference between these two projects?

**Comment by stephanrauh** [comment]
Your bug fix has landed with version 22.0.0-alpha.8.

Enjoy!
Stephan
